src/graph_notebook/neptune/client.py

```diff
@@ -141,7 +164,7 @@ def do_sparql_request(self, data: dict, headers=None, explain: str = '', path: s
             sparql_path = f'/{path}'
         elif self.sparql_path != '':
             sparql_path = f'/{self.sparql_path}'
-        elif "amazonaws.com" in self.host:
+        elif self.is_neptune_domain():
```

Comment:
Could you also replace the same Neptune host check for Gremlin here:

https://github.com/aws/graph-notebook/blob/c9157a4df73384c74537f3d1019cb240767aef0b/src/graph_notebook/neptune/client.py#L193

Reply:
I did this originally, looks like it's a merge change. I'll go ahead and add back in